_docs/technical/2017-02-16-delegation.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: Stake Delegation in Cardano SL
+permalink: /technical/delegation/
+group: technical
+visible: true
+---
+[//]: # (Reviewed at e1d0f9fb37a3f1378341716916f0321fb55698df)
+
+# Stake Delegation in Cardano SL
+
+This guide describes implementation details of the stake delegation process.
+
+As described earlier, stakeholders must be online in order to generate new blocks when they are selected as slot-leaders.
+However, such a situation can be unattractive, because of requiring that a majority of elected stakeholders participate in the
+Coin Tossing protocol for refreshing randomness (crucial attribute of the slot-leader election process). But if we have a
```

Comment:
This whole paragraph is too informal. 
@staenrey please look this over as well.